values.yaml

```
@@ -104,6 +104,9 @@ global:
   # Enables namespace labels for network policies
   networkNSLabels: false
 
+  rootAdmin:
+    username: root@root.com
```

We should always use RFC compliant example domain names for examples. https://www.rfc-editor.org/rfc/rfc2606.html

```suggestion
    username: root@example.com
```